src/components/AIReviewStream/AIReviewStream.tsx

```diff
@@ -41,30 +39,34 @@ export function AIReviewStream(props: AIReviewStreamProperties): JSX.Element {
             console.log("No UUID for review stream");
             return;
         } else {
-            ai_socket.on('connect', onConnect);
+            ai_socket.on("connect", onConnect);
             ai_socket.on(uuid, onMessage);
             if (ai_socket.connected) {
                 onConnect();
             }
         }
 
         function onJwtChange() {
-            const user = data.get('config.user');
-            const user_jwt = data.get('config.user_jwt');
+            const user = data.get("config.user");
+            const user_jwt = data.get("config.user_jwt");
             if (!user.anonymous && user_jwt) {
-                ai_socket.send('authenticate', {jwt: user_jwt});
```

Comment:
I removed single-quotes from the config because there are a lot of double quotes too so I figured let prettier take the wheel.

(I actually prefer single quotes, so wouldn't be mad if others wanted to re-add that setting)

Reply:
I'm not really opinionated either way

Reply:
I'm not fussed either.   Where context allows (I mean in the absence of anyone else's obvious standards and no semantic difference) I've used the "double quotes for things people are intended to see, single for internal strings" thing, FWIW, but that kind of thing only works if everyone knows and does ... unlikely in a shared codebase like this.   

And maybe that points us in the direction of "keep it simple, clear, set a standard and stick to it" -> turn it on.